gems/openssl/CVE-2018-16395.yml

```diff
@@ -0,0 +1,43 @@
+---
+gem: openssl
+cve: 2018-16395
+ghsa: mmrq-6999-72v8
+url: https://www.ruby-lang.org/en/news/2018/10/17/openssl-x509-name-equality-check-does-not-work-correctly-cve-2018-16395/
+title: Incorrect value comparison in Ruby openssl
+date: 2018-10-17
+description: |
+  An issue was discovered in the OpenSSL library in Ruby when two OpenSSL::X509::Name
+  objects are compared using ==, depending on the ordering, non-equal objects may
+  return true. When the first argument is one character longer than the second, or
+  the second argument contains a character that is one less than a character in the
+  same position of the first argument, the result of == will be true. This could be
+  leveraged to create an illegitimate certificate that may be accepted as legitimate
+  and then used in signing or encryption operations.
+cvss_v2: 7.5
+cvss_v3: 9.8
+patched_versions:
+  - ">= 2.0.9"
```

Comment:
The URL mentions that the openssl gem 2.1.2 was released with the security fix. Should be `>= 2.1.2`.

I've also submitted a correction to the GHSA advisory.

Reply:
Changed patched_version number.